templateflow/api.py

```diff
@@ -83,3 +86,29 @@ def _datalad_get(filepath):
             api.get(str(filepath))
         else:
             raise
+
+
+def _s3_get(filepath):
+    from math import ceil
+    from tqdm import tqdm
+    import requests
```

Comment:
Should probably add `tqdm` and `requests` dependencies.

Reply:
Yes, just noticed after committing.